src/app/pages/faq/faq.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SharedModule } from '@shared/shared.module';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
+import { FaqComponent } from './faq.component';
+
+const COMPONENTS = [FaqComponent];
+
+@NgModule({
+  declarations: [...COMPONENTS],
```

Comment:
By the way, it isn't needed spread it, like in `SharedModule`
```suggestion
  declarations: [COMPONENTS],
```

Reply:
Once again, my mistake for copy-pasting :sweat_smile:

Reply:
Also fixed in 32a3833